tasks/jetpack-compose/app/src/main/java/live/ditto/compose/tasks/MainActivity.kt

```diff
@@ -39,6 +39,7 @@ class MainActivity : ComponentActivity() {
             Root()
         }
 
+        ditto!!.store["tasks"].find("isDeleted == true").evict()
```

Comment:
So when is this evicting? Every time the app starts and a new live query is created? That may be okay, just trying to think about this a bit.

Reply:
Yes, it evicts when the app starts up. Because of the simplicity of the app there's not many places they could evict. tow options would be when the app starts or every time they update the isDeleted value then we could just evict immediately after